view/loadView.js

```diff
@@ -306,9 +306,9 @@ module.exports = function loadView(app, passport) {
 
         // return token
         const token = jwtUtil.createToken(req.user.name, req.user.name);
+        req.session.token = token;
```

Comment:
So now if we don't need to send plain token back, can we remove /authenticate route and make use of /v1/authenticate instead?